### PR TITLE
Update S.IO.Comp.clrcompression-x64 nupkg to version 4.0.0-beta-22819

### PR DESCRIPTION
**Pull request by FiveTimesTheFun**
Update S.IO.Comp.clrcompression-x64 nupkg to version 4.0.0-beta-22819
and copy native binary from new package location to output directory.

### CODE REVIEW

File: src/System.IO.Compression/src/System.IO.Compression.csproj [hunk 1]
```diff
@@ -65,5 +65,11 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="$(PackagesDir)\System.IO.Compression.clrcompression-x64\4.0.0-beta-22819\native\win\x64\clrcompression.dll">
```
Comment (ericstj) [inline src/System.IO.Compression/src/System.IO.Compression.csproj]: You can use an asterisk here in place of package version.

**Comment by ericstj** [comment]
Lgtm

File: src/System.IO.Compression/src/System.IO.Compression.csproj (hunk 1 above)
Comment (FiveTimesTheFun) [inline src/System.IO.Compression/src/System.IO.Compression.csproj]: What will it do then if multiple versions are on the box?

**Comment by FiveTimesTheFun** [comment]
Thanks Eric :)

File: src/System.IO.Compression/src/System.IO.Compression.csproj (hunk 1 above)
Comment (ericstj) [inline src/System.IO.Compression/src/System.IO.Compression.csproj]: Last one would win.  I don't care so much since this goes away in a week or so.

**Pull request merged**